frontend/apps/marketing/cicd/3-app/deploy.rb

```diff
@@ -274,12 +302,12 @@ def get_stack_output(stack_name, output_key, region)
     }
 
     deploy_stack(
-      cert_stack_name,
-      cert_template_path,
-      cert_parameters,
-      "us-east-1",
-      options[:role_arn],
-      options[:environment_type]
+      stack_name: cert_stack_name,
+      template_file: cert_template_path,
+      parameters: cert_parameters,
+      region: 'us-east-1',
+      role_arn: options[:role_arn],
+      tags: {'environment-type' => options[:environment_type], 'site-type' => options[:site_type]}
```

Comment:
Missing new line here?

```suggestion
      tags: {'environment-type' => options[:environment_type], 
      'site-type' => options[:site_type]}
```

Reply:
I thought this hash was short enough to be a one-liner, but standard multi-line, indented seems better!